tests/common/libvm.sh

```diff
@@ -98,8 +98,8 @@ EOF
 
   vm_setup
 
-  # XXX: hack around https://github.com/systemd/systemd/issues/14328
-  vm_cmd systemctl mask --now systemd-logind
+  # # XXX: hack around https://github.com/systemd/systemd/issues/14328
+  # vm_cmd systemctl mask --now systemd-logind
```

Comment:
Looks like this works now then? Feel free to just delete these lines entirely.

Reply:
Oh man, my bad. I think I've been debugging these bits of code for so long my eyes just got fatigued, leading to the bunch of errors :crying_cat_face: